### DISCUSSION TITLE
License: MIT/BSD vs GPL

### DISCUSSION THREAD

**Discussion by EliteScientist** [obsproject/obs-studio#8368] [Development Questions] [answered]
Hello,

I was looking into using this project, completely changing the UI, and adding some features. The problem I have is that, if I used this project, it is under the GPL license. This means that if I develop a commercial application that I would have to release the source code to it.

I love contributing to Open Source products and always contribute bug fixes, features, and etc... But wouldn't want to release any proprietary IP created for our own purpose.

Have you considered the MIT or BSD license?

**Comment by Fenrirthviti** [comment, marked as answer]
The GPL currently fits our ideology the best, and provides protections for us as a large project that the MIT and BSD licenses do not.

**Reply by jp9000** [reply]
Furthermore, it also enforces freedoms for the users that MIT and BSD do not.

**Reply by EliteScientist** [reply]
Understandable.
